angular-client/index.js

```diff
@@ -70,7 +70,35 @@ app.controller('login' , function ($scope,$http ) {
 		})
 	}
 })
-
+app.controller('project' , function ($scope,$http ) {
+	var get = function () {
+	var get={
+		method:"GET",
+		url : '/login'
+	}
+		$http(get).then(function (data) {
```

Comment:
Indentation in this whole section needs to be improved. The `$http(get)` call should be at the same level of indentation as `var get = {`. Missing some semicolons as well.